common/beerocks/tlvf/src/include/beerocks/tlvf/beerocks_message.h

```diff
@@ -139,7 +175,15 @@ class message_com {
             return nullptr;
         }
 
-        return add_intel_vs_data<T>(cmdu_tx, tlvhdr, id);
+        auto beerocks_header = add_intel_vs_data<T>(tlvhdr, id);
+        cmdu_tx.header = beerocks_header;
```

Comment:
Why isn't this done in add_intel_vs_data?

Reply:
Obsolete and now it is.